### PR TITLE
en,zh: add release note 1.3.3

### PR DESCRIPTION
**Pull request by KanShiori**
### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: en/releases/release-1.3.3.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+Release date: May 16, 2022
+
+TiDB Operator version: 1.3.3
+
+## New Feature
+
+- Add a new field `spec.tidb.service.port` to customize tidb service port ([#4512](https://github.com/pingcap/tidb-operator/pull/4512), [@KanShiori](https://github.com/KanShiori))
+
+## Bug fixes
```
Comment (ran-huang) [inline en/releases/release-1.3.3.md:15]:
```suggestion
## Bug fix
```

File: en/releases/release-1.3.3.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+- Fix the issue evict leader scheduler may leak when upgrading cluster ([#4522](https://github.com/pingcap/tidb-operator/pull/4522), [@KanShiori](https://github.com/KanShiori))
```
Comment (ran-huang) [inline en/releases/release-1.3.3.md:17]:
```suggestion
- Fix the issue that evict leader scheduler may leak during cluster upgrade ([#4522](https://github.com/pingcap/tidb-operator/pull/4522), [@KanShiori](https://github.com/KanShiori))
```

Comment (ran-huang) [inline en/releases/release-1.3.3.md:17]:
```suggestion
- Fix the issue that evict leader scheduler may leak during cluster upgrade ([#4522](https://github.com/pingcap/tidb-operator/pull/4522), [@KanShiori](https://github.com/KanShiori))
```

File: en/releases/release-1.3.3.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+# TiDB Operator 1.3.3 Release Notes
+
+Release date: May 16, 2022
+
+TiDB Operator version: 1.3.3
+
+## New Feature
+
+- Add a new field `spec.tidb.service.port` to customize tidb service port ([#4512](https://github.com/pingcap/tidb-operator/pull/4512), [@KanShiori](https://github.com/KanShiori))
```
Comment (ran-huang) [inline en/releases/release-1.3.3.md:13]:
```suggestion
- Add a new field `spec.tidb.service.port` to customize the tidb service port ([#4512](https://github.com/pingcap/tidb-operator/pull/4512), [@KanShiori](https://github.com/KanShiori))
```

Comment (ran-huang) [inline en/releases/release-1.3.3.md:13]:
```suggestion
- Add a new field `spec.tidb.service.port` to customize the tidb service port ([#4512](https://github.com/pingcap/tidb-operator/pull/4512), [@KanShiori](https://github.com/KanShiori))
```

**Review by ran-huang — COMMENTED** [review]
Rest LGTM

File: en/releases/release-1.3.3.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+# TiDB Operator 1.3.3 Release Notes
+
+Release date: May 16, 2022
+
+TiDB Operator version: 1.3.3
+
+## New Feature
+
+- Add a new field `spec.tidb.service.port` to customize the tidb service port ([#4512](https://github.com/pingcap/tidb-operator/pull/4512), [@KanShiori](https://github.com/KanShiori))
```
Comment (DanielZhangQD) [inline en/releases/release-1.3.3.md:13]: Please also add RN for 4498, 4490, 4500

Comment (DanielZhangQD) [reply en/releases/release-1.3.3.md:13]: https://github.com/pingcap/tidb-operator/pulls?q=is%3Apr+milestone%3Av1.3.3+is%3Aclosed

Comment (KanShiori) [reply en/releases/release-1.3.3.md:13]: Updated [2aedd6d](https://github.com/pingcap/docs-tidb-operator/pull/1826/commits/2aedd6d967168a3d7b977a1c0303f7366aca8fc8)

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Comment by KanShiori** [comment]
@ran-huang PTAL again

File: en/releases/release-1.3.3.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+## Bug fixes
+
+- Fix the issue that evict leader scheduler may leak during cluster upgrade ([#4522](https://github.com/pingcap/tidb-operator/pull/4522), [@KanShiori](https://github.com/KanShiori))
+
+- Update the base image of `tidb-backup-manager` to fix incompatibility with ARM architecture ([#4490](https://github.com/pingcap/tidb-operator/pull/4490), [@better0332](https://github.com/better0332))
+
+- Fix the issue that operator may panic when tidb Service don't have any Endpoints ([#4500](https://github.com/pingcap/tidb-operator/pull/4500), [@mikechengwei](https://github.com/mikechengwei))
```
Comment (ran-huang) [inline en/releases/release-1.3.3.md:21]:
```suggestion
- Fix the issue that TiDB Operator may panic when tidb Service does not have any Endpoints ([#4500](https://github.com/pingcap/tidb-operator/pull/4500), [@mikechengwei](https://github.com/mikechengwei))
```

File: zh/releases/release-1.3.3.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+- 添加新的 `spec.tidb.service.port` 字段，以支持配置 tidb 服务端口 ([#4512](https://github.com/pingcap/tidb-operator/pull/4512), [@KanShiori](https://github.com/KanShiori))
+
+## Bug 修复
+
+- 修复集群升级过程中，evict leader scheduler 可能泄漏的问题 ([#4522](https://github.com/pingcap/tidb-operator/pull/4522), [@KanShiori](https://github.com/KanShiori))
+
+- 更新 `tidb-backup-manager` 镜像的基础镜像，以修复不兼容 ARM 架构的问题 ([#4490](https://github.com/pingcap/tidb-operator/pull/4490), [@better0332](https://github.com/better0332))
+
+- 修复当 tidb Service 没有 Endpoint 时，operator 可能会 panic 的问题 ([#4500](https://github.com/pingcap/tidb-operator/pull/4500), [@mikechengwei](https://github.com/mikechengwei))
```
Comment (ran-huang) [inline zh/releases/release-1.3.3.md:21]:
```suggestion
- 修复当 tidb Service 没有 Endpoint 时，TiDB Operator 可能会 panic 的问题 ([#4500](https://github.com/pingcap/tidb-operator/pull/4500), [@mikechengwei](https://github.com/mikechengwei))
```

File: en/releases/release-1.3.3.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+- Fix the issue that Labels and Annotations of component Pods may lost after retrying access Kubernetes server ([#4498](https://github.com/pingcap/tidb-operator/pull/4498), [@duduainankai](https://github.com/duduainankai))
```
Comment (ran-huang) [inline en/releases/release-1.3.3.md:23]:
```suggestion
- Fix the issue that Labels and Annotations of the component Pods may be lost after TiDB Operator fails to access the Kubernetes server and retries ([#4498](https://github.com/pingcap/tidb-operator/pull/4498), [@duduainankai](https://github.com/duduainankai))
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**